src/applications/common/schemaform/definitions/address.js

```diff
@@ -73,8 +73,9 @@ const requiredFields = ['street', 'city', 'country', 'state', 'postalCode'];
  * @param {object} schema - Schema for a full form, including address definition in definitions
  * @param {boolean} isRequired - If the address is required or not, defaults to false
```

Comment:
what is `addressProperty`

Reply:
Added an additional JsDoc comment below.